fragments/filter/README.md

```diff
@@ -0,0 +1,632 @@
+                  [43.6079,-79.4893],
+                  [43.5677,-79.4632],
+                  [43.6129,-79.3925],
+                  [43.6223,-79.3238],
+                  [43.6576,-79.3163],
+                  [43.7945,-79.1178],
+                  [43.8144,-79.1542],
+                  [43.8555,-79.1714],
+                  [43.7509,-79.6390],
+                  [43.5845,-79.5442]
+              ]
+          ]
+        }
+      ]
+    ]
+  }
+}
+```
+
+### Example 3
+
+Queryable properties can be used on either side of an operator. This is a generic example, as there are are few STAC properties
+that are comparable in a meaningful way. This example uses a contrived example of two proprietary properties, `prop1` and `prop2` that are of the 
+same type. 
+
+This queryables JSON Schema is used in these examples:
+
+```json
+{
+  "$schema" : "https://json-schema.org/draft/2019-09/schema",
+  "$id" : "https://example.org/queryables",
+  "type" : "object",
+  "title" : "Queryables for Example STAC API",
+  "description" : "Queryable names for the example STAC API Item Search filter.",
+  "properties" : {
+    "prop1" : {
+      "description" : "Property 1",
+      "type": "integer"
+    },
+    "prop2" : {
+      "description" : "Property 2",
+      "type": "integer"    
+    }
+  }
+}
+```
+
+#### Example 3: GET with cql-text
+
+```http
+GET /search?filter=prop1 = prop2
+```
+
+#### Example 3: POST with cql-json
+
+```http
+POST /search
+{ 
+  "filter-lang"="cql-json",
+  "filter": {
+    "eq": [
+      { "property": "prop1" },
+      { "property": "prop2" }
+    ]
+  }
+}
+```
+
+### Example 4
+
+We'll be imagining these as queries against [EarthSearch Sentinel 2 
+COG](https://stacindex.org/catalogs/earth-search#/Cnz1sryATwWudkxyZekxWx6356v9RmvvCcLLw79uHWJUDvt2?t=items)' data.
+
+The queryables defined are as follows:
+
+```json
+{
+  "$schema" : "https://json-schema.org/draft/2019-09/schema",
+  "$id" : "https://example.org/queryables",
+  "type" : "object",
+  "title" : "Queryables for Example STAC API",
+  "description" : "Queryable names for the example STAC API Item Search filter.",
+  "properties" : {
+    "geometry" : {
+      "description" : "Geometry",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#/geometry"
+    },
+    "datetime" : {
+      "description" : "Datetime",
+      "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/datetime.json#/properties/datetime"
+    },
+    "eo:cloud_cover" : {
+      "description" : "Cloud Cover",
+      "$ref": "https://stac-extensions.github.io/eo/v1.0.0/schema.json#/properties/eo:cloud_cover"
+    },
+    "sentinel:data_coverage" : {
+      "description" : "Sentinel Data Coverage",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 100
+    }
+  }
+}
+```
+
+Note that `sentinel:data_coverage` is a proprietary extension (e.g., not defined in a formal, public way), and must be defined directly within the schema.
```

Comment:
I think the wording here could use tweaking. I don't think 'proprietary; means 'not defined in a formal, public way'. I think we can just call it an 'unspecified extension' or something? I do think we also want to have data providers define schemas, even if they aren't in a public, stac schemas extension. Like ideally there's a sentinel:data_coverage schema that sentinel defines. I know we're not there yet, but I think it could be good to make it seem like more of a special case to not have a schema. Like say that when STAC extensions have not yet defined their fields in a JSON Schema they can directly define it in queryables, but maybe make it a more 'custom' example than sentinel.

Reply:
tweaked